tests/getaddrinfo_test.c

```diff
@@ -0,0 +1,29 @@
+
+#include <netdb.h>
+#include <stdio.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+
+#include "greatest/greatest.h"
+
+TEST test_huggingface()
+{
+   struct addrinfo* res;
+   int rc = getaddrinfo("huggingface.co", "443", NULL, &res);
```

Comment:
We are depending on getaddrinfo() to use sendmmsg() and recvmmsg() to test these interfaces?
So if they change getaddrinfo() at some point in the future these interfaces aren't being tested.

Reply:
Yes I was, so I added a new test to exercise `sendmsg`, `recvmsg`, `sendmmsg`, and `recvmmsg` explicitly due to this well deserved chiding. Please see `tests/sendmsg_test.c`.